presto-product-tests/README.md

```diff
@@ -102,12 +102,12 @@ The Presto product tests must be run explicitly because they do not run
 as part of the Maven build like the unit tests do. Note that the product
 tests cannot be run in parallel. This means that only one instance of a
 test can be run at once in a given environment. To run all product
-tests and exclude the `quarantine` and `big_query` groups run the
+tests and exclude the `quarantine`, `big_query` and `profile` groups run the
 following command:
 
 ```
 ./mvnw install -DskipTests
-presto-product-tests/bin/run_on_docker.sh <profile> -x quarantine,big_query,mysql_connector,postgresql_connector
+presto-product-tests/bin/run_on_docker.sh <profile> -x quarantine,big_query,mysql_connector,postgresql_connector,profilespecifictests
```

Comment:
Don't break up this line. This is an example where you can break the 80 character limit. The Github Markdown rendering engine will wrap the line appropriately.